test/evergreen.yml

```diff
@@ -3430,7 +3429,183 @@ tasks:
             # Record the start time, in seconds
             date +%s > ../time.txt
             # Perform the tests to get code coverage metrics for
```

Comment:
It'd be nice to have a comment describing the philosophy/criteria used for the test selection (for our future selves).